vignettes/run-simulation.Rmd

```diff
@@ -127,9 +130,12 @@ observer <- getQuantity("Organism|Lumen|Aciclovir|Fraction dissolved", sim)
 addOutputs(c(molecule, observer), simulation = sim)
 
 # Add new outputs as path strings
-addOutputs(c("Organism|Lumen|Stomach|Aciclovir", 
-             "Organism|PeripheralVenousBlood|Aciclovir|Whole Blood (Peripheral Venous Blood)"),
-           simulation = sim)
+addOutputs(c(
+  "Organism|Lumen|Stomach|Aciclovir",

```

Comment:
Same here. In this case, this is always equivalent. But this is not always the case that changing the order does not matter

Reply:
I _think_ I made this change for the same reason as I above.